src/io/flutter/view/TimeDilationAction.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.view;
+
+import com.intellij.icons.AllIcons;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import io.flutter.run.daemon.FlutterApp;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class TimeDilationAction extends FlutterViewToggleableAction {
```

Comment:
I would think you need to extend FlutterViewAction instead of FlutterViewToggleableAction for this to be safe.

Reply:
I believe this extends FlutterViewToggleableAction because it needs access to isSelected(), which is not part of FlutterViewAction.